module/modmanager/manager.go

```diff
@@ -40,6 +41,9 @@ import (
 	rutils "go.viam.com/rdk/utils"
 )
 
+// tcpPortRange is the beginning of the port range. Only used when ViamTCPSockets() = true.
+const tcpPortRange = 13500
```

Comment:
What happens when a port is occupied?

Reply:
when a port is occupied, this breaks; long term, we need a better way to do this. can potentially request an unused port (`:0`) and have module report to RDK via the parent socket, but that's more plumbing

Reply:
I'm perfectly cool with this. I'd recommend copying that expectation into the documentation:
```
tcpPortRange is the beginning of the port range. Only used when ViamTCPSockets() = true. If a port is already taken, the server breaks and behaves in an undefined way. Given this is for a pre-production windows release, we're ok with this outcome. The code/algorithm will become more robust in the future.
```

Reply:
Why isn't the above showing up with word/line wrapping?
![image](https://github.com/user-attachments/assets/bfa1e57c-ca55-47f1-9a2f-57ffa46cf738)